### PR TITLE
Copilot - Customer Preview

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request creates a beta article for Copilot.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-38](https://sumologic.atlassian.net/browse/DOCS-38)

### CODE REVIEW

File: docs/search/copilot.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+:::tip
+Because manually typing an AI prompt requires careful precision for optimal performance, we recommend clicking the prebuilt [Suggestions](#suggestions-recommended) prompts, which have been proven effective through extensive testing.
+:::
+
+In the **Ask Something...** field, enter a natural language query prompt similar to the ones under **Suggestions** > **Explore**.
+
+You'll need to be very specific. Broad questions do not return good results. When your question is framed as a query about a small, well-defined problem, Copilot answers more accurately.
+
+:::note
+If the statement in the **Ask Something** field can't be translated into a query, this field will say "Failed translation".
```
Comment (devSpartan6631) [inline docs/search/copilot.md:86]: We are missing `...` (three periods) here.

File: docs/search/copilot.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+### Show/Hide Log Query
+
+Click **Show Log Query** or **Hide Log Query** to show the current investigation as a log query, or hide the query.
+
+<img src={useBaseUrl('img/search/copilot/show-hide-query.gif')} alt="Copilot time period" style={{border: '1px solid gray'}} width="500" />
+
+### Explore
+
+Click any of the **Suggestions** > **Explore** prebuilt suggestions to apply them to your investigation.
```
Comment (devSpartan6631) [inline docs/search/copilot.md:103]: Explore suggestions on the right updates only when `_sourceCategory` is modified. Explore suggestions do not represent the refinements ove executed query.

File: docs/search/copilot.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+## Step 4: Refine your investigation
+
+After executing a prompt, you'll see your current investigation summarized in plain text in the **Ask Something...** field. You can use these natural language query prompt ideas to launch and/or refine investigations.
+
+Optionally, follow any of the below steps to refine your search.
+
+### Show/Hide Log Query
+
+Click **Show Log Query** or **Hide Log Query** to show the current investigation as a log query, or hide the query.
```
Comment (devSpartan6631) [inline docs/search/copilot.md:97]: IMO: Show/Hide Log Query is not beneficial until we modify the query to refine our results. Should we merge this section with `Edit query code` ? WDYT ?

**Review by devSpartan6631 — COMMENTED** [review]
Some minor comments

File: docs/search/copilot.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Click **Select Source Category** and select the data source of the log messages you want to investigate. This field is also called the source expression box.
+
+<img src={useBaseUrl('img/search/copilot/source-category.png')} alt="Copilot source category" style={{border: '1px solid gray'}} width="600" />
+
+## Step 3: Execute a prompt
+
+### Suggestions (recommended)
+
+Under **Suggestions** > **Explore**, click on any of the suggested prompts to refine your investigation. For example:<br/><img src={useBaseUrl('img/search/copilot/suggestions-explore.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
```
Comment (kimsauce) [inline docs/search/copilot.md:73]:
```suggestion
Under **Suggestions** > **Explore**, click on any of the suggested prompts to start your investigation. For example:<br/><img src={useBaseUrl('img/search/copilot/suggestions-explore.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
```

File: docs/search/copilot.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+:::tip
+Because manually typing an AI prompt requires careful precision for optimal performance, we recommend clicking the prebuilt [Suggestions](#suggestions-recommended) prompts, which have been proven effective through extensive testing.
+:::
+
+In the **Ask Something...** field, enter a natural language query prompt similar to the ones under **Suggestions** > **Explore**.
+
+You'll need to be very specific. Broad questions do not return good results. When your question is framed as a query about a small, well-defined problem, Copilot answers more accurately.
+
+:::note
+If the statement in the **Ask Something** field can't be translated into a query, this field will say "Failed translation".
```
Comment (kimsauce) [inline docs/search/copilot.md:86]:
```suggestion
If the statement in the **Ask Something...** field can't be translated into a query, this field will say "Failed translation".
```

File: docs/search/copilot.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+<img src={useBaseUrl('img/search/copilot/source-category.png')} alt="Copilot source category" style={{border: '1px solid gray'}} width="600" />
+
+## Step 3: Execute a prompt
+
+### Suggestions (recommended)
+
+Under **Suggestions** > **Explore**, click on any of the prebuilt suggested prompts to start your investigation. For example:
+
+<img src={useBaseUrl('img/search/copilot/suggestions-explore.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
```
Comment (kimsauce) [inline docs/search/copilot.md:75]:
```suggestion
<img src={useBaseUrl('img/search/copilot/explore.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
```

File: docs/search/copilot.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/search/copilot.md]: Good call! Fixed.

File: docs/search/copilot.md (hunk 2 above)
Comment (kimsauce) [reply docs/search/copilot.md]: Ah that makes sense. Just edited.

File: docs/search/copilot.md (hunk 3 above)
Comment (kimsauce) [reply docs/search/copilot.md]: Done! I agree.

File: docs/search/copilot.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+## Step 1: Open Copilot
+
+To start using Copilot, navigate to the **Copilot** tab on the Sumo Logic home page.
+
+<img src={useBaseUrl('img/search/copilot/copilot-tab.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="250" />
+
+## Step 2: Select a source category
+
+Click **Select Source Category** and select the data source of the log messages you want to investigate. This field is also called the source expression box.
```
Comment (AnvitagGarg) [inline docs/search/copilot.md:65]: Shouldn't it be enter/select as user might want to enter his source category instead of selecting ?

**Comment by jpipkin1** [comment]
@kimsauce, I reviewed this PR, and it looks great! Awesome work! If I could approve the PR, I would. But I can't since I reassigned it to you. The good news is that you can approve it yourself.

My only comment is that the video needs audio to walk the viewer through it. Maybe you can ask @banant about that if you agree.

**Comment by kimsauce** [comment]
> @kimsauce, I reviewed this PR, and it looks great! Awesome work! If I could approve the PR, I would. But I can't since I reassigned it to you. The good news is that you can approve it yourself.
> 
> 
> 
> My only comment is that the video needs narration to walk the viewer through it. Maybe you can ask @banant about that if you agree it needs narration.

Thank you, John! I manually wrote out the steps underneath the video, since it has no audio. I will make a note about missing audio as well so the user isn't confused.

File: docs/search/copilot.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+        title="Copilot Demo"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+Scenario: We receive a Sumo Logic alert about AWS CloudTrail access keys being leaked outside our organization and use Copilot to investigate.
```
Comment (kimsauce) [inline docs/search/copilot.md:45]: Add note above this stating the steps below are meant to replace the audio. (Video has no audio)

File: docs/search/copilot.md (hunk 7 above)
Comment (kimsauce) [reply docs/search/copilot.md]: Fixed - let me know what you think

File: docs/search/copilot.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
+---
+id: copilot
+title: Sumo Logic Copilot - Feature Preview
+sidebar_label: Copilot 🤖
```
Comment (kimsauce) [inline docs/search/copilot.md:4]:
```suggestion
sidebar_label: Copilot
```

**Review by kimsauce — APPROVED** [review]
Publishing as an unlisted doc per @banant

**Pull request merged**